src/Core/Form/IdentifiableObject/DataHandler/CategoryFormDataHandler.php

```diff
@@ -26,103 +26,24 @@
 
 namespace PrestaShop\PrestaShop\Core\Form\IdentifiableObject\DataHandler;
 
-use PrestaShop\PrestaShop\Core\CommandBus\CommandBusInterface;
+use PrestaShop\PrestaShop\Core\Domain\Category\Command\AbstractAddCategoryCommand;
+use PrestaShop\PrestaShop\Core\Domain\Category\Command\AbstractEditCategoryCommand;
 use PrestaShop\PrestaShop\Core\Domain\Category\Command\AddCategoryCommand;
 use PrestaShop\PrestaShop\Core\Domain\Category\Command\EditCategoryCommand;
-use PrestaShop\PrestaShop\Core\Domain\Category\ValueObject\CategoryId;
-use PrestaShop\PrestaShop\Core\Image\Uploader\ImageUploaderInterface;
-use Symfony\Component\HttpFoundation\File\UploadedFile;
 
 /**
  * Creates/updates category from data submitted in category form
  */
-final class CategoryFormDataHandler implements FormDataHandlerInterface
+class CategoryFormDataHandler extends AbstractCategoryFormDataHandler
```

Comment:
Why not keep it final?

Reply:
Must have been an accident added it back.